### ISSUE TITLE
🔥[Regression] 6.3.2 breaks collectionGroup queries

### ISSUE THREAD

**Issue by harrisrobin** [invertase/react-native-firebase#3196]
## Issue

Ever since upgrading to 6.3.2, collectionGroup queries with `.where` have broken in my app.
Tested it using 6.3.1, I don't face this issue on that version.

Here's the error I get when I attempt the following code:
```js
 const bookings = await db
      .collectionGroup('someCollection')
      .where("user", "==", userRef)
      .get()
```

![image](https://user-images.githubusercontent.com/3880008/74304777-00783880-4d2c-11ea-88ec-bf31fbf02fa9.png)

My hunch is it might be caused by : https://github.com/invertase/react-native-firebase/commit/2cb6d44b77051f3831ed52b2687ce254d407904d

```

Exception '*** +[FIRFieldPath<0x102624cb8> init]: cannot init a class object.' was thrown while invoking collectionGet on target RNFBFirestoreCollectionModule with params (
    "[DEFAULT]",
    bookings,
    collectionGroup,
        (
                {
            fieldPath =             (
                user
            );
            operator = EQUAL;
            value =             (
                11,
                "users/....userId"
            );
        }
    ),
        (
    ),
        {
    },
    "<null>",
    8826,
    8827
)
```
---

### Javascript

<details><summary>Click To Expand</summary>
<p>

#### `package.json`:

```json
# N/A
```

#### `firebase.json` for react-native-firebase v6:

```json
# N/A
```

### iOS

<details><summary>Click To Expand</summary>
<p>

#### `ios/Podfile`:

- [ ] I'm not using Pods
- [x] I'm using Pods and my Podfile looks like:

```ruby
# N/A
```

#### `AppDelegate.m`:

```objc
// N/A
```

</p>
</details>

---

## Environment

<details><summary>Click To Expand</summary>
<p>

**`react-native info` output:**

```
System:
    OS: macOS 10.15.4
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Memory: 634.80 MB / 32.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.11.1 - ~/.nvm/versions/node/v12.11.1/bin/node
    Yarn: 1.21.1 - /usr/local/bin/yarn
    npm: 6.11.3 - ~/.nvm/versions/node/v12.11.1/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      Android NDK: 17.2.4988734
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.6010548
    Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
```

- **Platform that you're experiencing the issue on**:
  - [x] iOS
  - [ ] Android
  - [x] **iOS** but have not tested behavior on Android
  - [ ] **Android** but have not tested behavior on iOS
  - [ ] Both
- **`react-native-firebase` version you're using that has this issue:**
  - `e.g. 6.3.2`
- **`Firebase` module(s) you're using that has the issue:**
  - `e.g. Instance ID`
- **Are you using `TypeScript`?**
  - `Y` & `3.7.5`
  
</p>
</details>

**Comment by Ehesp** [comment]
Yep this was my fault sorry. Fix is already in place just sorting a couple of other things then will release

**Comment by Ehesp** [comment]
This is a general problem with where queries too, not just related to collection group ones

**Comment by Ehesp** [comment]
https://github.com/invertase/react-native-firebase/pull/3192

**Comment by russellwheatley** [comment]
@harrisrobin #3192 this has been fixed and released into v6.3.3. Thanks for raising the issue 👍
